### DISCUSSION TITLE
Program stretches when running in version 0.3.8 or higher.

### DISCUSSION THREAD

**Discussion by jshsakura** [EricGoldsteinNz/tadpole#55] [General]
[xfgle.hgp]
0  *
0  *
0  *
0  *
2 콘트라 (Contra).zfc*
2 타카하시 명인의 모험도 (Takahashi Meijin no Bouken-jima).zfc*
2 칩과 데일의 대작전 (Chip to Dale no Daisakusen).zfc*
2 더블 드래곤 3 - 로제타 스톤 (Double Dragon III The Rosetta Stone).zfc*
3 파이널 판타지 6 (Final Fantasy 6).zsf*
3 소닉 윙즈.zsf*
3 별의 커비 3 (Hoshi no Kirby 3).zsf*
3 초 마계촌.zsf*
4 소닉 3D 블래스트 (Sonic 3D Blast ~ Sonic 3D Flickies' Island).zmd*
4 코믹스 존.zmd*
4 시노비 III - 닌자 마스터의 귀환.zmd*
4 스트리트 파이터 II 스페셜 챔피언 에디션.zmd*
5 젤다의 전설 꿈꾸는 섬 (Legend of Zelda, The - Link's Awakening).zgb*
5 키드 이카루스 - 신화와 괴물.zgb*
5 테트리스.zgb*
5 바이오닉 코만도.zgb*
5 아스트릭스 서치 포 도그메틱스.zgb*
5 블레이드.zgb*
5 봄버맨 셀렉션 (Bomberman Selection).zgb*
5 함타로 - 햄-햄 단결! (En).zgb*
5 포켓몬스터 - 에메랄드 (Pocket Monsters - Emerald).zgb*
5 파이어 엠블렘 (Fire Emblem).zgb*
5 다운타운 열혈물어 EX (Downtown - Nekketsu Monogatari EX).zgb*
5 록맨 제로 (Rockman Zero).zgb*
6 mslug.zip*
6 dino.zip*
6 samsho.zip*
6 kof97.zip*

[error]
Result Finished updating FC with 97 ROMs
loading roms to table for (D:\) (FC)
Traceback (most recent call last):
  File "C:\Users\GPD\Documents\tadpole\tadpole.py", line 438, in reloadDriveList
    self.combobox_drive_change()
  File "C:\Users\GPD\Documents\tadpole\tadpole.py", line 662, in combobox_drive_change
    RunFrogTool(newDrive,console)
  File "C:\Users\GPD\Documents\tadpole\tadpole.py", line 91, in RunFrogTool
    window.loadROMsToTable()
  File "C:\Users\GPD\Documents\tadpole\tadpole.py", line 1261, in loadROMsToTable
    position = tadpole_functions.getGameShortcutPosition(drive, system, game)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\GPD\Documents\tadpole\tadpole_functions.py", line 496, in getGameShortcutPosition
    lines = xfgle_file_handle.readlines()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeDecodeError: 'cp949' codec can't decode byte 0xeb in position 32: illegal multibyte sequence

[why?]
I am using the shortcut file by editing it myself. (UTF-8)
Normal use was not possible if the modified shortcut was included.
In the end I had to delete xfgle.hgp.

**Comment by EricGoldsteinNz** [comment]
Thank you for finding this issue. I can uploaded a fix now to use UTF-8 for shortcuts. 

I will need to add this to other areas of the application as well I think, but for now can you please test that this fixes the issue you had?

You may need to grab a clean xfgle.hgp from https://github.com/EricGoldsteinNz/SF2000_Resources/blob/main/OS/V1.6/Resources/xfgle.hgp

**Reply by jshsakura** [reply]
Thank you so much.
I will check and let you know as soon as I get off work.!
